scripts/homestead.rb

```diff
@@ -328,6 +328,7 @@ def self.configure(config, settings)
         load_balancer = settings['load_balancer'] ||= false
         http_port = load_balancer ? '8111' : '80'
         https_port = load_balancer ? '8112' : '443'
+        site_php = settings.has_key?('php') ? settings['php'] : '8.1'
```

Comment:
I guess `||=` would work here. The above lines don't suggest a style preference. I wasn't sure if it would throw an undefined of some kind. Don't write enough Ruby.